free-2.md

```diff
@@ -58,12 +75,20 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I think that the project would be split between language design and systems work. It would entail the construction of a formula that takes many different parameters and can simulate behavior over time. There would ideally be a non-trivial graphical display portion of the project. The language would need to allow for the creation of new variables and parameters. The DSL could be made more complex if it gave users more options (i.e. the ability to choose different variables to graph against one another). 
+
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+This is a big idea and an ambitious project, though I think that it's achievable in the timeframe of the class. It involves a lot of different UI and language design components and has the potential to be a very useful tool.
+
 
 ### Benefits and drawbacks
 _Why might this be a good idea for a project? Why might this not be a good idea 
 project?_
 
+This project could appeal to a wide variety of people and be expanded to different types of beverages. It would be user friendly and technically challenging for the developer. It might not be a good project because the DSL might be relatively simple and overshadowed by the brewing formula. This would depend on the implementation of the DSL and the end capabilities of the project.
```

Comment:
As I mentioned earlier, the brewing formula to create simulation itself seems to be both a core aspect of the project yet not exactly the most relevant to DSLs.